test/e2e/fixture/app/consequences.go

```diff
@@ -52,15 +51,7 @@ func (c *Consequences) app() *Application {
 }
 
 func (c *Consequences) get() (*Application, error) {
-	output, err := fixture.RunCli("app", "get", c.context.name, "-o", "yaml")
-	if err != nil {
-		if strings.Contains(output, "NotFound") {
-			return nil, nil
-		}
-		return nil, err
-	}
-	app := &Application{}
-	return app, yaml.Unmarshal([]byte(output), app)
+	return fixture.AppClientset.ArgoprojV1alpha1().Applications(fixture.ArgoCDNamespace).Get(c.context.name, v1.GetOptions{})
```

Comment:
I know this looks odd, but the intent to was make sure that we prefer to invoke the CLI tool, than the API. How come you changed this bit?

Reply:
Forgot to mention it in PR description. This method is executed most frequently during e2e tests execution. Looks like any process fork takes at least 100ms, so I decided to switch it to api call to save several seconds